transferui/src/test/java/com/hyperwallet/android/ui/transfer/viewmodel/CreateTransferViewModelTest.java

```diff
@@ -306,7 +322,7 @@ public void testCreateQuoteTransfer_isSuccessfulWithAllAvailableFunds() {
         viewModel.setTransferNotes("Create quote test notes");
         viewModel.setTransferAllAvailableFunds(true);
         viewModel.setTransferAmount("124.23");
-
+        viewModel.init();
```

Comment:
don't you want to verify which load is being called?

Reply:
FYI: Discussed. Implemented in some other method.